src/hearings/containers/request-hearing/welsh-hearing/welsh-hearing.component.html

```diff
@@ -0,0 +1,35 @@
+<exui-error-message *ngIf="error" [error]="error"></exui-error-message>
+
+<div>
+  <form class="form" [formGroup]="welshForm" [ngClass]="{'govuk-form-group--error': error}">
+    <exui-hearing-parties-title></exui-hearing-parties-title>
+    <h1 class="govuk-heading-l">
+        Does this hearing need to be in Welsh?
+    </h1>
+    <div id="welsh-hint" class="govuk-hint">
+        This means the entire hearing will be carried out in Welsh.
+    </div>
+
+    <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset">
+            <span id="welsh-hearing-location" [ngClass]="{'govuk-error-message': error}" *ngIf="error">
+              <span class="govuk-visually-hidden">Error:</span> Select hearing need to be in Welsh or not
+            </span>
+
+            <div class="govuk-radios govuk-radios--inline" data-module="govuk-radios">
+                <div class="govuk-radios__item">
+                    <input class="govuk-radios__input" id="welsh_hearing_yes" name="hearingInWelshFlag" formControlName="hearingInWelshFlag" type="radio" [value]="true">
+                    <label class="govuk-label govuk-radios__label" for="welsh_hearing_yes">Yes</label>
+                </div>
+                <div class="govuk-radios__item">
+                    <input class="govuk-radios__input" id="welsh_hearing_no" name="hearingInWelshFlag" formControlName="hearingInWelshFlag" type="radio" [value]="false">
+                    <label class="govuk-label govuk-radios__label" for="welsh_hearing_no">No</label>
+                </div>
+            </div>
+
+        </fieldset>
+    </div>
+  </form>
+  
+  <button class="govuk-button" data-module="govuk-button" (click)="onSubmit()">Continue</button>
```

Comment:
Can this 'Continue' button move to the parent component?

Reply:
Moved to parent component